src/bundle.c

```diff
@@ -172,28 +172,37 @@ static int unload_tracked_bundle(const char *bundle_name, struct list **subs)
 }
 
 /* Return list of bundles that include bundle_name */
-struct list *required_by(const char *bundle_name, struct manifest *mom)
+void required_by(struct list **reqd_by, const char *bundle_name, struct manifest *mom, int rl)
```

Comment:
I'd personally prefer a different, more descriptive name than r1, even though there is a comment below. Up to you on that :)

Reply:
yeah yeah, done. ;)